### DISCUSSION TITLE
Hosted instance?

### DISCUSSION THREAD

**Discussion by M-Gonzalo** [bottomless-archive-project/library-of-alexandria#321] [General]
Hi! I have a question regarding the contents of this project. Is it available to the world in some website, or is everyone supposed to build their own local copy?

**Comment by laxika** [comment]
Unfortunately, for legal reasons I don't think showing my instance is possible, at least in my country. Maybe it might happen later on for selected members, I need to look into the legal repercussions. :(

For now, you are supposed to build your own instance.

Maybe we might come up with a better solution later on? I'm no lawyer, but creating something peer-to-peer could work. :D
